### PR TITLE
Update FirstFragment.kt

### PR DESCRIPTION
**Pull request by firstSupakorn**

### CODE REVIEW

**Comment by firstSupakorn** [comment]
vcdfdsf

**Comment by jnjtnutty** [comment]
reject

**Comment by firstSupakorn** [comment]
> reject

File: app/src/main/java/com/example/journey/FirstFragment.kt [hunk 1]
```diff
@@ -13,6 +13,7 @@ import androidx.navigation.fragment.findNavController
  */
 class FirstFragment : Fragment() {
     val test123 = "123test"
+    val test1jklll = ""
```
Comment (jnjtnutty) [inline app/src/main/java/com/example/journey/FirstFragment.kt:16]: change variable name

**Pull request closed without merging**